packages/react-native/src/primitives/Checkbox/Checkbox.tsx

```diff
@@ -34,12 +31,14 @@ export default function Checkbox<T>({
   value,
   ...rest
 }: CheckboxProps<T>): JSX.Element {
+  const [pressed, setPressed] = useState(selected ?? false);
```

Comment:
Should `Checkbox` be a controlled or uncontrolled component, or optionally both? ([RWA CheckboxField](https://ui.docs.amplify.aws/react/components/checkboxfield#controlled-component))